open-hackathon-server/src/hackathon/hack/team_manager.py

```diff
@@ -161,6 +161,61 @@ def get_tema(team):
 
         return [get_tema(x) for x in teams]
 
+    def get_admin_hackathon_team_list(self, hackathon_id, name=None, number=None):
```

Comment:
Where is the pagination? Suppose the method should accept `page_number`, `page_index` or similar